resources/assets/sass/effect/trein.scss

```diff
@@ -29,6 +29,7 @@
 .ns-slt-6 { background: url("../../images/trein/slt_6.png") 0 0; }
 .ns-sng-4 { background: url("../../images/trein/sng_4.png") 0 0; }
 .rnet-gtw { background: url("../../images/trein/gtw_e_8_rnet.png") 0 1px; }
+.thalys { background: url("../../images/trein/thalys.png") 0 0; }
```

Comment:
Het plaetje staat in de `resources/assets/images` map, deze moet verplaatst worden.